buildkite/DOCKER_DEPLOY_ENV

```diff
@@ -0,0 +1,4 @@
+CODA_SERVICE=bot
+CODA_GIT_TAG=$BUILDKITE_COMMIT
+CODA_GIT_BRANCH=$BUILDKITE_BRANCH
+CODA_DEB_REPO=$BUILDKITE_BRANCH
```

Comment:
At least in the existing implementation of the docker upload script, there are other env vars that are added as part of earlier build steps (ask @lk86 for more details)